src/actions/is-equal.ts

```diff
@@ -0,0 +1,21 @@
+import 'colors';
+import * as _ from 'lodash';
+import Action from '../action.i';
+
+/**
+ * Checks two given sets of values for equality.
+ */
+export default class IsEqual implements Action {
+  constructor() {}
```

Comment:
Please check comments on previous file, most of them apply here as well.

Reply:
Done